test/DynamoCoreTests/Logging/AnalyticsServiceTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Dynamo.Configuration;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Nodes.ZeroTouch;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.Scheduler;
+using DynamoShapeManager;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using SystemTestServices;
+using TestServices;
+
+namespace Dynamo.Tests.Loggings
+{
+    [TestFixture]
+    class AnalyticsServiceTest : DynamoModelTestBase
+    {
+        //We need to override this function because the one in DynamoModelTestBase is setting StartInTestMode = true
+        protected override DynamoModel.IStartConfiguration CreateStartConfiguration(IPreferences settings)
+        {
+            return new DynamoModel.DefaultStartConfiguration()
+            {
+                PathResolver = pathResolver,
+                StartInTestMode = false,
+                GeometryFactoryPath = preloader.GeometryFactoryPath,
+                Preferences = settings,
+                ProcessMode = TaskProcessMode.Synchronous
+            };
+        }
+
+        /// <summary>
+        /// This test method will validate that the AnalyticsService.OnWorkspaceAdded (CustomNodeWorkspaceModel) is executed
```

Comment:
@RobertGlobant20 Is there other event tested? for example any node related ones?

Reply:
@QilongTang no, i think this is the only event in workspace (or nodes) tested as a part of my test cases added.